apps/andi/mix.exs

```diff
@@ -39,12 +39,14 @@ defmodule Andi.MixProject do
       {:divo, "~> 1.1", only: [:dev, :integration]},
       {:divo_kafka, "~> 0.1.5", only: [:dev, :integration]},
       {:divo_redis, "~> 0.1.4", only: [:dev, :integration]},
+      {:floki, "~> 0.23"},
```

Comment:
Should this be scoped to `[:test, :integration]` ?

Reply:
Floki is pretty small and doesn't have any running components, so I think it should be fine to leave here. Do we have much of a pattern either way?

Reply:
As far as I know we've been pretty diligent about making sure things like `placebo`, `checkov`, `smartcities_test`, etc. are test and integration.

Reply:
I scoped it to dev, testing, and integration